TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -134,19 +126,20 @@ formats in `R`.
 ## Exploring Date and Time Classes
 
 ###R - Date Class - as.Date
-We can use the`as.Date` method to convert a date field in string/character 
-format to an `R` `Date` class. It will ignore all values after the date string.
+As we just saw, when we use the`as.Date` method to convert a date field in string/character 
+format to an `R` `date` class, it will ignore all values after the date string.
+Let's explore date and time classes in more detail separately from our data set.
 
 ```{r explore-as_date }
 #Convert character data to date (no time) 
 myDate <- as.Date("2015-10-19 10:15")   
 str(myDate)
-head(myDate)
+myDate
```

Comment:
it's nice to use head rather than trying to spit back the entire object - more efficient but also renders nicer when you knit the lessons. I'd put head(myDate) back in there unless there is a string justification otherwise!